docs/changelog/changelog_en.md

```diff
@@ -1,49 +1,46 @@
 # Changelog plentyshopPWA
 
-## v1.x.x (yyyy-mm-dd)
-
-- Added tax and shipping note in wishlist page
-- Added new carousel compoment via Swipper library
-- Created new useCarousel composable holding Carousel logic
-- Minified and purged swiper css files for speed optimization
-- Fluid logo container + max-width and max-height addaptation
-- Fix cls mobile
-- Edit mode toolbar + Json editor Front End
+## v1.7.0 (2024-11-06) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.6.0...v1.7.0" target="_blank" rel="noopener"><b>Overview of all changes</b></a>
 
 ### New
 
-- Clicking "Buy" with an unsaved Address will now display a notification.
-- When an EU country is selected as the shipping country, all EU countries become available as billing options.
-- Added a new request header for configId and added no cache to environment variables.
-- Implement new notification design
-- Adding the ability to have alt text for images.
-- It's now possible to change the font color of the hero banner via a template property.
-- Display product titles without global suffix when available, maintaining backward compatibility.
+- Clicking the **Buy** button with an unsaved address now displays a notification.
+- Selecting an EU country as the shipping country now makes all EU countries available as billing options.
+- Improved the product title display for SEO purposes. Reverts to the default if no product specific title is available.
```

Comment:
This is actually a new field available on the items' data. This means you can either have a custom title or it is using the old pattern of "Item name | Company name"